.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+### Before requesting gatekeeper:
+
+* [ ] Two review approvals and any prior change requests have been resolved.
+* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it
+      should be force landed and forced-landing tag is set.
+* [ ] Commit messages follows the guidelines outlined
+      [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
+* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.
+
+### Gatekeeper:
+
+* [ ] You are the appropriate gatekeeper to be landing the patch.
+* [ ] The PR has 2 reviews by people familiar with the code.
+* [ ] Any appropriate watchers have had a chance to review the PR.
+* [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested
+      changes.
+* [ ] Githooks were used. If not, there is a sufficient reason to move forward.  Check copyrights
+      if githooks were not used.
```

Comment:
We really should add a proper CI check for copyright dates, and we should do it before the next change-of-year.